src/v2/components/ManageChannel/index.tsx

```diff
@@ -0,0 +1,349 @@
+              </Field>
+
+              <Accordion label="NSFW?" mode="closed">
+                <Field
+                  name="content_flag"
+                  initialValue={channel.content_flag.toUpperCase()}
+                >
+                  {props => {
+                    return (
+                      <Box m={7}>
+                        <Text f={2} mb={7}>
+                          Not Safe For Work (NSFW) channels are hidden from
+                          Explore and are only visible on your profile to people
+                          who have the &quot;Show NSFW Content&quot; setting
+                          turned on.
+                        </Text>
+                        <RadioOptions
+                          value={props.input.value}
+                          onSelect={props.input.onChange}
+                          size="1em"
+                        >
+                          <RadioOptions.Option value="SAFE">
+                            {({ selected }) => (
+                              <Text f={3} mb={3} selected={selected}>
+                                <strong>Safe for work</strong>
+                              </Text>
+                            )}
+                          </RadioOptions.Option>
+
+                          <RadioOptions.Option value="NSFW">
+                            {({ selected }) => (
+                              <Text f={3} mb={3} selected={selected}>
+                                <strong>Not safe for work</strong>
+                              </Text>
+                            )}
+                          </RadioOptions.Option>
+                        </RadioOptions>
+                      </Box>
+                    )
+                  }}
+                </Field>
+              </Accordion>
+
+              {channel.can.export && (
+                <Accordion label="Export" mode="closed">
+                  <Box m={7}>
+                    <ExportChannel id={channel.id} />
+                  </Box>
+                </Accordion>
+              )}
+
+              {channel.can.transfer && (
+                <Accordion label="Transfer ownership" mode="closed">
+                  <Box m={7}>
+                    <TransferChannel channel={channel} />
+                  </Box>
+                </Accordion>
+              )}
+
+              {channel.can.destroy && (
+                <Accordion label="Delete channel" mode="closed">
+                  <Box m={7}>
+                    <DeleteChannel
+                      id={channel.id}
+                      onDelete={onDelete}
+                      refetchQueries={refetchQueries}
+                    />
+                  </Box>
+                </Accordion>
+              )}
+            </Container>
+          </TitledDialog>
+        )
+      }}
+    />
+  )
+}
+
+interface ManageChannelContainerProps {
+  id: string | number
+  refetchQueries?: PureQueryOptions[]
+  onUpdate?: () => any
+  onDelete?: () => any
```

Comment:
generally you want to return void if you're ignoring the callback's return value